entproto/cmd/protoc-gen-entgrpc/template.go

```diff
@@ -68,3 +71,23 @@ func (t tmplValues) retrieve(token string) (interface{}, error) {
 	}
 	return k, nil
 }
+
+type wrappedCalls struct {
```

Comment:
The name `calls` is confusing because conversion is not a function call. Do you find it clearer than this?

```go
g.Tmpl("v := %(a)(%(b))", tmplValues{
	"a": protogen.GoImportPath("golang.google.com/protobuf/wrapperspb").Ident("Int32"),
	"b": "int32(x)",
})
```

Reply:
yes because we need to support 3 cases in this template:
1. ent field type == pb field type - no conversion/function
2. simple conversion - ( int32(1) )
3. two step conversion wrapperspb.Int32(int32(1))
having this struct that can represent all cases makes the template and passing the instructions around the code cleaner

maybe if we call it `typeConversion`, and the member field is called `ops` or something like that?

Reply:
> 1. ent field type == pb field type - no conversion/function

Why do you need it in this case, and how do you use it in this example.

> 2. simple conversion - ( int32(1) )
> 3. two step conversion wrapperspb.Int32(int32(1))
>    having this struct that can represent all cases makes the template and passing the instructions around the code cleaner
> 
> maybe if we call it `typeConversion`, and the member field is called `ops` or something like that?

Not sure. The first one is a type conversion and the second one is a function call. Maybe let's just make it more explicit for the use-case that you use in the template (do you have any examples to share? maybe in another PR)? Also, `arguments` should be a single `argument`, right?